### PR TITLE
n8n: support community nodes

### PR DESCRIPTION
**Pull request by Yann-J**
## Proposed changes

This adds support for custom community nodes to n8n (as per https://docs.n8n.io/integrations/community-nodes/)
This works by creating an `initContainer` for both `master` and `worker` nodes that will run npm install to install each node from the list given in the `communityNodes` Helm value.

I send this PR this because I've created a custom node this weekend: https://www.npmjs.com/package/@oneacrefund/n8n-nodes-infobip

## Checklist

- [ ] Lint and unit tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added documentation to describe my feature.
- [ ] I have squashed my commits into logic units.
- [ ] I have described the changes in the commit messages.

## Other information

One weird thing is that I've seen we are persisting the `/root/.n8n` directory as data folder. However, [the docs](https://hub.docker.com/r/n8nio/n8n#persist-data) indicate that the app is using `/home/node/.n8n` - I'm not sure why that is, and if that's expected...

### CODE REVIEW

File: charts/n8n/values.yaml [hunk 1]
```diff
@@ -236,54 +239,3 @@ postgresql:
   postgresqlDatabase: n8n
   persistence:
     size: "200Gi"
-
-gatekeeper:
```
Comment (nzeugaa) [inline charts/n8n/values.yaml:240]: Why removing `gatekeeper` ?

Comment (Yann-J) [reply charts/n8n/values.yaml:240]: I noticed it wasn't used by the chart, so assumed it was a leftover that needed cleanup...

Comment (bisonlou) [reply charts/n8n/values.yaml:240]: Yes, We can have one separate gatekeeper chart that we install for any service, by simply overriding values.

Comment (nzeugaa) [reply charts/n8n/values.yaml:240]: deploying gatekeeper via helm chart - https://open-policy-agent.github.io/gatekeeper/website/docs/install/#deploying-via-helm

**Review by nzeugaa — APPROVED** [review]

**Comment by nzeugaa** [comment]
@bisonlou merging this PR, we will need to edit the n8n release to add a step installing gatekeeper, and also adding `oneacrefund/n8n-nodes-infobip` as a community node to be installed

**Pull request merged**